File: pdlearning/src/microservices/calendar-microservice/Api/Application/Models/TeamMemberEventOverviewModel.cs
using System;

namespace Microservice.Calendar.Application.Models
{
    public class TeamMemberEventOverviewModel
    {
        public Guid Id { get; set; }

        public string Title { get; set; }

        public DateTime StartAt { get; set; }

        public DateTime EndAt { get; set; }
    }
}

File: pdlearning/src/web-apps/src/libraries/domain-api/src/calendar/models/team-member-event-overview.model.ts
export interface ITeamMemberEventOverviewModel {
  id: string;
  title: string;
  startAt: Date;
  endAt: Date;
}

export class TeamMemberEventOverviewModel implements ITeamMemberEventOverviewModel {
  public id: string;
  public title: string;
  public startAt: Date;
  public endAt: Date;

  constructor(data: ITeamMemberEventOverviewModel) {
    this.id = data.id;
    this.title = data.title;
    this.startAt = data.startAt;
    this.endAt = data.endAt;
  }
}
